docs/source/releases/latest/764-integration-tests-pass-without-noaa-data.yaml

```diff
@@ -0,0 +1,12 @@
+bug fix:
```

Comment:
Wait to merge until 1.14.0 is tagged/released.

Reply:
switched to dev-staging, so can merge now